library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -137,32 +134,34 @@ class ChuckerInterceptor @JvmOverloads constructor(
     /**
      * Processes a [ResponseBody] and populates corresponding fields of a [HttpTransaction].
      */
-    private fun processResponseBody(response: Response, responseBody: ResponseBody?, transaction: HttpTransaction) {
-        getNativeSource(response)?.use { source ->
-            source.request(java.lang.Long.MAX_VALUE)
-            val buffer = source.buffer()
-            var charset: Charset = UTF8
-            val contentType = responseBody?.contentType()
-            if (contentType != null) {
-                try {
-                    charset = contentType.charset(UTF8) ?: UTF8
-                } catch (e: UnsupportedCharsetException) {
-                    return
-                }
+    private fun processResponseBody(response: Response, responseBody: ResponseBody, transaction: HttpTransaction) {
```

Comment:
Since you started changes here, will drop a small concern.
I don't understand why can't we just pass only `response` here and retrieve `responseBody` inside this fun? 

Or another possible change - if we look for `response` usages we might see that is used only for headers check. So, maybe it is worth to pass just headers into this fun?

Reply:
> I don't understand why can't we just pass only response here and retrieve responseBody inside this fun?

Yeah we could. Should we make this change in another PR?